Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -332,6 +337,13 @@ private extension EncodedAppleReceipt {
             return content.asData.hashString
         case let .receipt(data):
             return data.hashString
+        case .sk2receipt(let receipt):
```

Comment:
Nit for consistency with the rest:
```suggestion
        case let .sk2receipt(receipt):
```